app/views/trainees/_filters.html.erb

```diff
@@ -21,31 +21,33 @@
     <% end %>
   <% end %>
 
-  <div class="govuk-form-group">
-    <fieldset class="govuk-fieldset">
-      <legend class="govuk-fieldset__legend govuk-fieldset__legend--s">
-        <% if trainee_search_path?(search_path) %>
-          <%= t("views.trainees.index.filters.record_completion") %>
-        <% else %>
-          <%= t("views.drafts.index.filters.record_completion") %>
-        <% end %>
-      </legend>
-      <div class="govuk-checkboxes govuk-checkboxes--small">
-        <% [:complete, :incomplete].map do |completion_status| %>
-          <div class="govuk-checkboxes__item">
-            <%= check_box_tag "record_completion[]",
-                              completion_status,
-                              checked?(filters, :record_completion, completion_status.to_s),
-                              id: "record_completion-#{completion_status}",
-                              class: "govuk-checkboxes__input" %>
-            <%= label_tag "record_completion-#{completion_status}",
-                          label_for("record_completion", completion_status),
-                          class: "govuk-label govuk-checkboxes__label" %>
-          </div>
-        <% end %>
-      </div>
-    </fieldset>
-  </div>
+  <% unless FeatureService.enabled?(:user_can_have_multiple_organisations) && session[:current_organisation][:type] == "School" %>
```

Comment:
I'm don't think we need to check whether the feature is on here.

Also re `session[:current_organisation][:type] == "School"` - we seem to have a couple of different ways we're checking for this currently. Foe example, @jhenderson has repurposed the `hide_draft_records?` helper in PR https://github.com/DFE-Digital/register-trainee-teachers/pull/2081. 

Related: https://github.com/DFE-Digital/register-trainee-teachers/pull/2081#pullrequestreview-896278458

Might just need a quick chat with John so that you're using the same logic in the same place 👍